lib/src/components/NxToggle/stateful/__tests__/NxStatefulCheckbox.test.tsx

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import NxToggle from '../../NxToggle';
+import NxStatefulToggle, { Props } from '../NxStatefulToggle';
+import {getShallowComponent} from '../../../../__testutils__/enzymeUtils';
+
+describe('NxStatefulToggle', function() {
+  const simpleProps: Props = {
+    toggleId: 'toggle-id',
+    defaultChecked: false,
+    onChange: () => {},
+    disabled: undefined,
+    children: undefined
```

Comment:
This should probably have a value

Reply:
> This should probably have a value

Done in both tests: https://github.com/sonatype/sonatype-react-shared-components/pull/250/commits/17fa230b2cb7c56a1ba9c642d7e9bef8c3c005f5